studio/stores/authConfig/schema/AuthProviders/AuthProvidersFormValidation.tsx

```diff
@@ -95,7 +96,7 @@ const PROVIDER_PHONE = {
       title: 'Twilio Account Sid',
       show: {
         key: 'SMS_PROVIDER',
-        matches: 'twilio',
+        matches: ['twilio'],
```

Comment:
Looks like a breaking change. Not sure why this is an array now.

Reply:
Hm previously, we were rendering certain form fields only if they matched certain providers. However,  with Twilio Verify, the OTP related fields (see screenshot below) should be excluded for Twilio Verify as Twilio Verify manages the OTP expiry etc and any changes to OTP related fields would have no effect. 


As such, I modified the `matches` field to take in an array of providers that a field should render for instead of a single `provider` in order to specify that the OTP fields should only render for non-Twilio Verify Providers. Not too familiar with the codebase so open to suggestions on how this could be done better as well.

![CleanShot 2023-07-01 at 11 01 16@2x](https://github.com/supabase/supabase/assets/8011761/8df145b3-70e7-4d47-b6ab-6071ef68786b)